src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -998,4 +1096,20 @@ protected String getProtoRpcFullMethodName(Service protoService, Method protoMet
     return String.format(
         "%s.%s/%s", protoService.protoPakkage(), protoService.name(), protoMethod.name());
   }
+
+  protected TypeNode getTransportOperationsStubType(Service service) {
+    TypeNode transportOpeationsStubType = service.operationServiceStubType();
+    if (transportOpeationsStubType == null) {
+      transportOpeationsStubType = getTransportContext().transportOperationsStubTypes().get(0);
+    } else {
+      transportOpeationsStubType =
```

Comment:
I'm not sure I can read the suggested change right (it abruptly ends with empty return statement). I agree that initializing something and than unavoidably overwriting it may be confusing. Here I think it was done to save a little bit of code, because on the else statement the new value is constructed based on the old value and with current logic we can do `transportOpeationsStubType.reference()` instead of `service.operationServiceStubType().reference()` which is one less method call and it is being done twice in the else statement.